<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createpost</class>
 <widget class="QDialog" name="createpost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(4, 51, 255, 172);</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 38, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>shared</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>feeling</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>thinking_about</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>making</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>celebrating</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(4, 51, 255, 169);</string>
   </property>
   <property name="text">
    <string>Select Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(4, 51, 255, 169);</string>
   </property>
   <property name="text">
    <string>Add Activity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(4, 51, 255, 169);</string>
   </property>
   <property name="text">
    <string>Add Description:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>501</width>
     <height>74</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>681</width>
     <height>74</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="share">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid;
Border-radius: 10px;
background-color: rgba(0, 249, 0, 207);</string>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
